applications/user-api/src/main/java/no/fdk/userapi/UsersController.java

```diff
@@ -0,0 +1,49 @@
+package no.fdk.userapi;
+
+import no.fdk.altinn.AltinnClient;
+import no.fdk.altinn.Organisation;
+import no.fdk.altinn.Person;
+import no.fdk.webutils.exceptions.NotFoundException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/users")
+public class UsersController {
+    private static final Logger logger = LoggerFactory.getLogger(UsersController.class);
+
+    AltinnClient altinnClient;
```

Comment:
Is it possible to use "@Autowired" annotation here?

Reply:
med Autowire trenger man å bruke Spring i unit testene (som vi ikke har her enda). Uten autowire kan man kalle constructor med mock